src/DIRAC/FrameworkSystem/private/standardLogging/Logging.py

```diff
@@ -325,63 +342,81 @@ def getAllPossibleLevels():
     """
     return LogLevels.getLevelNames()
 
-  def always(self, sMsg, sVarMsg=''):
+  def always(self, sMsg, sVarMsg='', local_context=None):
```

Comment:
You still want to remove these again, right?

Reply:
oh ! something went wrong, I removed it for sure, I must have screwed my squash....